mscore/prefsdialog.cpp

```diff
@@ -82,6 +82,9 @@ PreferenceDialog::PreferenceDialog(QWidget* parent)
       if (CocoaBridge::isSystemDarkModeSupported())
             styleName->addItem(tr("System"));
 #endif
+#ifdef Q_OS_WIN
+      styleName->addItem(tr("System"));
```

Comment:
Note: this looks like an added string, but due to same context should reuse the existing translation, so really is not something that needs to get translated again